tools/nvqpp/nvq++.in

```diff
@@ -170,16 +170,19 @@ fi
 CUDAQ_IS_APPLE=@CUDAQ_IS_APPLE@
 INCLUDES=""
 LINKER_FLAGS=
+MACOS_LD_FLAGS=
 NVQPP_LD_PATH=${NVQPP_LD_PATH:-"@NVQPP_LD_PATH@"}
 if [ -f "$NVQPP_LD_PATH" ]; then
 	LINKER_FLAGS="${LINKER_FLAGS} --ld-path=${NVQPP_LD_PATH}"
 fi
-LINKER_FLAGS="${LINKER_FLAGS} -Wl,-rpath,${install_dir}/lib -Wl,-rpath,${PWD} -L${install_dir}/lib"
```

Comment:
Hmm. Why remove this from everything except MacOS?

I think it's safer to assume that if someone went through the trouble of adding something, they had a reason for doing it and it wasn't just a mistake. On the other hand, I don't know why these flags were introduced. They seem to be from the earliest prototype. (shrug)

Reply:
Sorry for the noise, I was using the CI to do a small test because I don't have a Linux environment set up (: